language/move-lang/tests/move_check/parser/invalid_character_non_ascii.move

```diff
@@ -0,0 +1,6 @@
+address 0x0:
+
+// Check that we provide good error messages for non-ASCII characters.
+module Temp {
+   ф
```

Comment:
A test that checks that it works when the character is in a comment (probably the most likely scenario?) might also be useful.

Reply:
At least for now, we intentionally disallow non-ASCII characters in comments (see https://github.com/libra/libra/pull/812), so I assume that by "it works" you mean that the compiler does not crash and produces a good error message. I added a test for that.

Reply:
Ah yes, that is indeed what I meant. Thanks for adding the test!